README.md

```diff
@@ -43,13 +47,17 @@ SnoVault JSON-LD Database Framework
 
 1. Java 8 (Java 11 has also been used)
     ```
-    $ brew tap AdoptOpenJDK/openjdk
-    $ brew cask install adoptopenjdk8
+    $ brew install adoptopenjdk8
     ```
 
 1. Elasticsearch 5.x
     ```
-    $ brew install elasticsearch@5.6
+    $ https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-5.6.0.tar.gz
+    $ tar -xvf ~/Downloads/elasticsearch-5.6.0.tar.gz -C /usr/local/opt/
```

Comment:
no ".gz"

Reply:
what do you mean?  The extension for the url and file is .tar.gz on the url and download?

Reply:
oh I think my browser automatically decompressed it.